### PR TITLE
PD 333 application restore fixes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/storage-operations/stateful-applications/crd-reference.md [hunk 1]
```diff
@@ -155,6 +158,8 @@ spec:
 |----|----|----|----|
 |backupName| The name of the ApplicationBackup you want to restore from | **Default:** None | Yes |
 |backupLocation| Which backup location object to get application backups from | **Default:** None | Yes |
+| namespaceMapping | A map of source and destination namespaces, allowing you to restore a backup to a different namespace. You must provide the map in key value pairs, with the source namespace as the key and the destination namespace as the value. {{<info>}}**NOTE:** You must run this spec from an admin namespace (kube-system by default). {{</info>}} | **Default:** None | No |
+| replacePolicy | What happens if matching resources already exist | **Default:** <br/><br/> **Enumerated string:** Delete, Retain |  |
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/stateful-applications/crd-reference.md]: @disrani-px Are these required? Is there a default value for replacePolicy?

Comment (disrani-px) [reply content/portworx-install-with-kubernetes/storage-operations/stateful-applications/crd-reference.md]: The default is Retain

Comment (nathan-px) [reply content/portworx-install-with-kubernetes/storage-operations/stateful-applications/crd-reference.md]: @disrani-px And is it required?

Comment (disrani-px) [reply content/portworx-install-with-kubernetes/storage-operations/stateful-applications/crd-reference.md]: No, it isn't required. If nothing is specified it'll default to Retain

**Pull request merged**